src/spec/Responses.php

```diff
@@ -42,7 +42,9 @@ public function __construct(array $data)
             // From Spec: This field MUST be enclosed in quotation marks (for example, "200") for compatibility between JSON and YAML.
```

Comment:
line 36 above (can't comment it directly) should be `@param Response[]|Reference[]|array`